scala/scala.bzl

```diff
@@ -164,6 +187,17 @@ def _write_test_launcher(ctx, jars):
       output=ctx.outputs.executable,
       content=content)
 
+def _collect_srcjars(targets):
+  srcjars = set()
+  for target in targets:
+    print(target) #TODO remove
+    #TODO perhaps we want to unify the naming?
+    if hasattr(target, "srcjar"):
```

Comment:
well, it's also listed in outputs for scrooge_scala_srcjar, but I don't think that that is ever made visible (it seems like outputs are not available to the targets graph, only attributes are)